src/components/CustomizableCalendarDay.jsx

```diff
@@ -160,14 +159,6 @@ export const selectedSpanStyles = {
   },
 };
 
-export const lastInRangeStyles = {
```

Comment:
to be really careful about breaking changes, we might want to keep this as `export const lastInRangeStyles = {}`

Reply:
I think making `lastInRangeStyles = {}` would fix the bug without needing to completely remove the modifier's styles. This would make it so that there is no built-in styling for `lastInRange`, thus fixing the bug, but you could still supply your own styles if you wanted to.